### ISSUE TITLE
Add an ability to look at hands and deck similar to graveyard

### ISSUE THREAD

**Issue by Svyatoslav28597** [magefree/mage#12576]
Currently, there is no way to check the deck or hands like in Arena, if something gets shuffled into the deck, like 3rd from the top - you can only keep this information in mind. If someone reveals a card from their hand - there is only an option of keeping window with a revealed card. I propose an improvement:
 You can look at the deck and hand at any time, just like the graveyard, however all cards are shown facedown. If a person reveals a card from their hand - everyone can see the face of the revealed card.  If you look at other players' hands with cards like thoughtseize - cards you looked at would be visible only to you. If something gets placed in a specific position in the deck (Approach of the Second sun, for example) - everyone can see the face of this card while looking at your deck.
 This may be somewhat hard to implement, but that would be a decent qol improvement which could also help with the reduction of visual clutter

**Comment by JayDi85** [comment]
Are you talking about opponent’s hand and deck?

**Comment by Svyatoslav28597** [comment]
Yes, about opponent's hands, however viewing both your and opponent decks could be useful, since there are cards that care about both [[Case the Joint]]  [[Approach of the second sun]] [[Deem Inferior]]

**Comment by JayDi85** [comment]
There are two different tasks:
1. Must show revealed/looked cards in all zones/windows (current logic uses special windows, so it required full rework of reveal/looked logic, see details in #12480);
2. Must show windows with hand and library on player panel click (current player can see own deck, but it’s not a library). It’s important to use fake objects data to hide it (see hidden data problems in face down topic #11881)

So it’s a hard task but possible

**Comment by tiera3** [comment]
Just seconding that this would be great.

A much simpler workaround, I've also mentioned previously that I would like it if a window that reveals a card isn't reused. That way you have the option of keeping the windows open to keep track of the revealed cards.
